storage/null.go

```diff
@@ -13,11 +13,9 @@ type NullStorage struct {
 }
 
 func (*NullStorage) PersistBatch(ctx context.Context, p ...model.Persistable) error {
-	log.Debugw("Not persisting data")
 	return nil
 }
 
 func (*NullStorage) PersistModel(ctx context.Context, m interface{}) error {
-	log.Debugw("Not persisting data")
 	return nil
 }
```

Comment:
These are pretty spammy... I wish there was something lower than "debug", like "trace". I don't want to see these but I will add them back if there are strong feelings about keeping them.